Client/src/Pages/PageSpeed/CreatePageSpeed/index.jsx

```diff
@@ -16,11 +18,12 @@ import Breadcrumbs from "../../../Components/Breadcrumbs";
 import "./index.css";
 
 const CreatePageSpeed = () => {
-	const MS_PER_MINUTE = 60000;
-	const { user, authToken } = useSelector((state) => state.auth);
-	const dispatch = useDispatch();
-	const navigate = useNavigate();
-	const theme = useTheme();
+  const MS_PER_MINUTE = 60000;
+  const { user, authToken } = useSelector((state) => state.auth);
+  const { isLoading } = useSelector((state) => state.uptimeMonitors);
```

Comment:
This is potentially problematic as we are using the loading state for the uptime monitors rather than page speed monitors.

I know it's kind of redundant, but I think it would be better to copy the action from the uptimeMonitorSlice to the pageSpeedMonitorSlice so that we're dealing with the correct loading state here.

Reply:
@ajhollid, If I understood correctly, instead of using state.uptimeMonitors I should use state.pageSpeedMonitors for Creating PageSpeed Monitors? Please correct me if I've misunderstood.